netdata-installer.sh

```diff
@@ -1020,17 +948,17 @@ if [ "${AUTOUPDATE}" = "1" ]; then
 			fi
 			progress "Installing new netdata-updater in cron"
 
-			rm ${installer_dir}/netdata-updater.sh || : #TODO(paulfantom): this workaround should be removed after v1.13.0-rc1. It just needs to be propagated
+			rm "${INSTALLER_DIR}/netdata-updater.sh" || : #TODO(paulfantom): this workaround should be removed after v1.13.0-rc1. It just needs to be propagated
```

Comment:
please check it exists before removing it.
this logs an error on console.

Reply:
> please check it exists before removing it.
> this logs an error on console.

Great catch, i guess just adding -f should do the trick unless we want to report the absence of the files

Reply:
Oh, right we are after v1.13.0-rc1 so I'll just delete the whole thing as TODO suggests.